networkx/algorithms/dag.py

```diff
@@ -56,8 +56,13 @@ def descendants(G, source):
     Examples
     --------
     >>> DG = nx.path_graph(5, create_using=nx.DiGraph)
-    >>> sorted(list(nx.descendants(DG, 2)))
-    [3, 4]
+    >>> descendants = nx.descendants(DG, 2)
+    >>> descendants
+    {3, 4}
+
+    A node is not returned as one of its own descendantsgit . If you prefer to include the node itself as well, you can perform a manual union operation as follows:
```

Comment:
It looks like `git` snuck itself in here :). A minor wording suggestion (feel free to ignore):

```suggestion
    The `source` node is not a descendant of itself, but can be included manually:
```